lib/streamlit/elements/alert.py

```diff
@@ -12,86 +12,147 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import cast, TYPE_CHECKING
+from typing import cast, Optional, TYPE_CHECKING
 
 from streamlit.proto.Alert_pb2 import Alert as AlertProto
+from streamlit.errors import StreamlitAPIException
+from streamlit.string_util import is_emoji_valid
 from .utils import clean_text
+import re
```

Comment:
nit: since this import is from the stdlib, it should be placed above the `from typing import ...` line

Reply:
Fixed on [f763899](https://github.com/streamlit/streamlit/pull/4964/commits/f763899713cc89a3d094995ef8915e9ca5c70424)